Makefile

```diff
@@ -74,13 +74,26 @@ ngrok:
 		--set credentials.apiKey="$(NGROK_AK)" \
 		--set credentials.authtoken="$(NGROK_AT)"
 
+.PHONY: tilt up destroy
```

Comment:
I'm not sold on having a `destroy` command available. I'd rather let people purposefully destroy their k3d cluster without it being hidden by an abstraction.

Reply:
This Make target creates the k3d cluster named "observability" if it doesn't exist. So, destroying the stack when done seems logical. It's purposely intended to be easy dedicated ephemeral stack that is easy to spin up and throw away.

Reply:
It's just as easy to say `echo "Creating k3d-observability cluster, make sure to delete it by running k3d cluster delete k3d-observability at your convience when done"` At best it could be a completely separate command.

Reply:
I removed the tilt-destroy target.